plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/OTelResourceProvider.java

```diff
@@ -24,9 +24,9 @@
 
 import java.util.concurrent.TimeUnit;
 
-import static org.opensearch.telemetry.OTelTelemetryPlugin.TRACER_EXPORTER_BATCH_SIZE_SETTING;
-import static org.opensearch.telemetry.OTelTelemetryPlugin.TRACER_EXPORTER_DELAY_SETTING;
-import static org.opensearch.telemetry.OTelTelemetryPlugin.TRACER_EXPORTER_MAX_QUEUE_SIZE_SETTING;
+import static org.opensearch.telemetry.OtelTelemetrySettings.TRACER_EXPORTER_BATCH_SIZE_SETTING;
+import static org.opensearch.telemetry.OtelTelemetrySettings.TRACER_EXPORTER_DELAY_SETTING;
+import static org.opensearch.telemetry.OtelTelemetrySettings.TRACER_EXPORTER_MAX_QUEUE_SIZE_SETTING;
 
 /**
  * This class encapsulates all OpenTelemetry related resources
```

Comment:
So we created a class and we do not use SpanExporterFactory is here: 

```
public static OpenTelemetry get(Settings settings) {
        return get(
            settings,
            SpanExporterFactory.create(),
            ContextPropagators.create(W3CTraceContextPropagator.getInstance()),
            Sampler.alwaysOn()
        );
}
```

Reply:
Ah, nice catch. I don't know how it got missed :)